### PR TITLE
Fix NatoUnarmed Loadout

### PR DESCRIPTION
**Pull request by Lazejun**
## What type of PR is this.
1. [x] Bug
2. [ ] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
Added Nato Unarmed in each Template including the Example,
Added the added Prefix to Natounarmed in fn_compatabilityLoadFaction
Added CSAT Unarmed in Compatfile

### Please specify which Issue this PR Resolves.
Not linked on purpose:
Solution 1 of #1834 

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Execute this on an Occupant Town, it should spawn Civies.
```sqf
//Target, Origin
//Targets are Cities in CitiesX
["Georgetown01", "airport_1"] spawn A3A_fnc_invaderPunish;
```
Creates Civs the same way as the Traitor Mission does right now, just in your Group
```sqf
 _civ = [group player, NATOUnarmed,player, [],0,"NONE"] call A3A_fnc_createUnit; 
 _civ forceAddUniform (selectRandom allCivilianUniforms); 
 _rnd = random 100; 
 if (_rnd < 90) then { 
  if (_rnd < 25) then { 
   [_civ, "hgun_PDW2000_F", 5, 0] call BIS_fnc_addWeapon; 
  } else { 
   [_civ, "hgun_Pistol_heavy_02_F", 5, 0] call BIS_fnc_addWeapon; 
  }; 
 };
 ```

********************************************************
Notes:

### CODE REVIEW

File: A3-Antistasi/Templates/NewTemplates/3CB/3CB_AI_TKA_East.sqf [hunk 1]
```diff
@@ -887,3 +887,7 @@ private _unitTypes = [
 ["other", [["Official", _policeTemplate]], _militaryLoadoutData] call _fnc_generateAndSaveUnitsToTemplate;
 //The following lines are determining the loadout for the AI used in the "kill the traitor" mission
 ["other", [["Traitor", _traitorTemplate]], _militaryLoadoutData] call _fnc_generateAndSaveUnitsToTemplate;
+//The following lines are determining the loadout for the AI used in the "Invader Punishment" mission
+["other", [["Unarmed", _UnarmedTemplate]], _militaryLoadoutData] call _fnc_generateAndSaveUnitsToTemplate;
+//The following lines are determining the loadout for the AI used in the "Invader Punishment" mission
+["other", [["Unarmed", _UnarmedTemplate]], _militaryLoadoutData] call _fnc_generateAndSaveUnitsToTemplate;
```
Comment (HakonRydland) [inline A3-Antistasi/Templates/NewTemplates/3CB/3CB_AI_TKA_East.sqf:893]: double unarmed loadout

File: A3-Antistasi/functions/Templates/fn_compatabilityLoadFaction.sqf [hunk 2]
```diff
@@ -242,7 +242,7 @@ if (_side isEqualTo west) then {
 	NATOOfficer2 = "loadouts_occ_other_Traitor";
 	NATOBodyG = "loadouts_occ_military_Rifleman";
 	NATOCrew = "loadouts_occ_other_Crew";
-	NATOUnarmed = "loadouts_occ_Unarmed";
+	NATOUnarmed = "loadouts_occ_other_Unarmed";
```
Comment (HakonRydland) [inline A3-Antistasi/functions/Templates/fn_compatabilityLoadFaction.sqf:245]: could probably add it for inv too

**Review by HakonRydland — CHANGES_REQUESTED** [review]
two minor changes

File: A3-Antistasi/functions/Templates/fn_compatabilityLoadFaction.sqf (hunk 2 above)
Comment (Lazejun) [reply A3-Antistasi/functions/Templates/fn_compatabilityLoadFaction.sqf:246]: Added with  https://github.com/official-antistasi-community/A3-Antistasi/pull/1851/commits/e0f861704c60e494e97dfe67e8fd984800ac0325

File: A3-Antistasi/Templates/NewTemplates/3CB/3CB_AI_TKA_East.sqf (hunk 1 above)
Comment (Lazejun) [reply A3-Antistasi/Templates/NewTemplates/3CB/3CB_AI_TKA_East.sqf]: Removed, thanks for point it out
https://github.com/official-antistasi-community/A3-Antistasi/pull/1851/commits/2c9eac49736947afa973ba948f1139fba4b6e9f9

**Review by HakonRydland — APPROVED** [review]
looks fine

**Pull request merged**